Update README.md (#1272)
vtexdocs/dev-portal-content: README.md

4. Send the PR in `#dev-portal-pr` Slack channel for approval.
5. After approval, merge the PR.

### How to remove an API reference from Developer Portal?

> You will need to create two pull requests in the following order: one for `devportal` repository and other for `openapi-schemas`.

Reviewer: ```suggestion
You will need to create two pull requests, the first one in [devportal](https://github.com/vtexdocs/devportal), and the second one in [openapi-schemas](https://github.com/vtex/openapi-schemas). Changes are detailed below.
```

> To remove the API content from `devportal` repository, follow the steps below:

Reviewer: ```suggestion
To remove the API content from `devportal`, follow the steps below:
```

1. Open a branch in the [devportal](https://github.com/vtexdocs/devportal) repository.
2. In the `src/pages/api/openapi/[slug].tsx` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.
3. In the `src/utils/getReferencePaths.ts` file, remove the content that refers to the API. It follows this syntax: `'{schemaFileName}': '{slug}'`.

> 4. Open a PR with the changes.

Reviewer: I suggest adding a step to remove the API and its endpoints from navigation.json

Reviewer: Maybe it's worth adding the redirect as an optional step here instead of in a callout in the end, since it's in the same repository

Author: Agree and changed it!

5. Send the PR in `#dev-portal-pr` Slack channel for approval.
6. After approval, merge the PR.

> Next, to remove the API content from [openapi-schemas](https://github.com/vtex/openapi-schemas) repository, follow the steps below:

Reviewer: ```suggestion
Next, to remove the API content from `openapi-schemas`, follow the steps below:
```

> 1. Open a branch in the `openapi-schemas` repository.

Reviewer: ```suggestion
1. Open a branch in the [openapi-schemas](https://github.com/vtex/openapi-schemas) repository.
```
